tests/fiaas_deploy_daemon/crd/test_crd_watcher.py

```diff
@@ -156,7 +151,6 @@ def test_is_able_to_watch_custom_resource_definition(self, crd_watcher, deploy_q
             (ADD_EVENT, "UPDATE", {"deployment": {"fiaas/source-repository": "xyz"}}, "xyz"),
             (MODIFIED_EVENT, "UPDATE", None, None),
             (MODIFIED_EVENT, "UPDATE", {"deployment": {"fiaas/source-repository": "xyz"}}, "xyz"),
-            (DELETED_EVENT, "DELETE", None, None),
```

Comment:
I feel that the test could be reworked and kept, to catch us not raising an exception. But it's a fair rewrite, admittedly. Perhaps make it a separate test.

Reply:
Regarding your previous comment, make sense to keep this test to be sure we are not raising an exception :+1:

Reply:
Created a dedicated test for delete event.
Let me know, if you think it's not enough.